### DISCUSSION TITLE
Can I hide or toggle sidebar?

### DISCUSSION THREAD

**Discussion by stefanos82** [zealdocs/zeal#1596] [Q&A] [answered]
When I read a long documenation, the sidebar distracts me.

Is there a way to either hide it or toggle it upon need?

**Comment by MohamedSaif44** [comment]
the work around is to open the page in the browser.

**Comment by trollixx** [comment, marked as answer]
Not at the moment but it is easy to implement. I'll add that for the next release.

Created issue #1652.
